Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -629,10 +629,7 @@ namespace OpenInfraPlatform {
 
 						if (!convertIfcFace(face, pos, polygon, polygonIndices)) 
 						{
-							std::stringstream text;
-							text << "IFC Face conversion failed with faces #" << faces.at(0)->getId() << "-" << faces.at(faces.size() - 1)->getId();
-
-							throw std::exception(text.str().c_str());
+							throw oip::UnhandledException(face); 

```

Comment:
This is not an `UnhandledException`, rather it is geometric failure.

Reply:
Resolved: a46e819